app/controllers/api/mixins/service_templates.rb

```diff
@@ -24,8 +24,11 @@ def api_request_allowed?
       end
 
       def request_from_ui?
-        return false if request.headers["x-auth-token"].blank?
-        token_info.present?
+        api_token = request.headers["x-auth-token"]
+        return false if api_token.blank?
+        token_info = Environment.user_token_service.token_mgr("api").token_get_info(api_token)
+        $api_log.info("XXXXXXXXXXX service_templates.rb:  The Token Used #{token_info}")
```

Comment:
This log line can be made less scary looking by removing the XXXX...

Reply:
My page was stale when I made that comment, looks like it's been fixed already

Reply:
yep, removed that log message :)